var/spack/repos/builtin/packages/py-nbconvert/package.py

```diff
@@ -13,21 +13,36 @@ class PyNbconvert(PythonPackage):
     homepage = "https://github.com/jupyter/nbconvert"
     url      = "https://github.com/jupyter/nbconvert/archive/4.2.0.tar.gz"
 
+    version('5.6.0', sha256='21a69fe5756a3af8e76dcc105217cc3bea4d8f23dbff5727d9fd1e4388b53beb')
     version('4.2.0', sha256='32394be5a20f39f99fabfb9b2f2625df17f1c2a6699182ca41598e98e7cc9610')
     version('4.1.0', sha256='459f23381411fd1ff9ec5ed71fcd56b8c080d97b3a1e47dae1c5c391f9a47266')
     version('4.0.0', sha256='00e25eeca90523ba6b774b289073631ef5ac65bb2de9774e9b7f29604516265c')
 
-    depends_on('py-pycurl', type='build')
-    depends_on('python@2.7:2.8,3.3:')
+    variant('execute', default=True, description='Include jupyter-client')
+    variant('serve', default=True, description='Include a webserver')
+
+    depends_on('py-pycurl', type='build',                         when='^python@2.7:2.8')
+    depends_on('py-setuptools', type='build',                     when='@5:')
+    depends_on('python@2.7:2.8,3.3:', type=('build', 'run'))
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'),      when='@5:')
     depends_on('py-mistune', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-mistune@0.8.1:1.999', type=('build', 'run'))
```
At least in the latest version

Reply:
I know, but it works without (at least for me) and we currently don't have a newer mistune version (fixed by #13887). I'm unsure whether to put the version restriction in here now or not

Reply:
Just put it in now, then when someone adds a newer version of mistune things don't break